ptf/tests/config/fdb_config.py

```diff
@@ -0,0 +1,56 @@
+import os
+import time
+import sys
+import inspect
+
+from config.constant import *
+from sai_thrift.sai_adapter import *
+from sai_base_utils import * # pylint: disable=wildcard-import; lgtm[py/polluting-import]
+
+class FdbT0Config(object):
+    """
+    Class use to make all the fdb configurations.
+    """
+    def create_fdb_mac_address_list(self, test):
+        test.mac_list = []
+        print("Generate mac address...")
+        test.mac0 = '00:00:00:00:00:11' 
+        test.mac_list.append(test.mac0)
+
+        for i in range(1, 32):  
+            tmp = i % 16
+            cnt = i // 16
+            str1 = '{:x}'.format(tmp) * 10
+            str2 = '{:02}'.format(cnt)
+            mac = str2 + str1
+            mac = ':'.join(mac[i:i+2] for i in range(0, len(mac), 2))
+            setattr(test, 'mac%s' % i, mac)
+            test.mac_list.append(mac)
+
+
+    def create_fdb_entries(self, test):
```

Comment:
follow the naming convention in config_t0.md to parameterize those functions for generating IPs, MACs, etc.